tests/rule-grouping/rule-grouping-1/README.md

```diff
@@ -0,0 +1,11 @@
+# Test Description
+
+Test to demonstrate the port grouping and SGH distribution for small range
+overlaps and single points with "any".
+
+## PCAP
+
+None
+
+## Related issues
+
```

Comment:
Did you leave ticket link out because this is still being discussed?

Reply:
No, because we don't have one but I think we should and that is still to be discussed. But, nonetheless, this PR should have been draft as it is incomplete. Thanks. :)